### ISSUE TITLE
investigate clone() in tfsemb_main.py

### ISSUE THREAD

**Issue by zkokaja** [hassonlab/247-pickling#101]
Is it actually needed? For logits and/or embeddings?
